spec/api-menu-spec.js

```diff
@@ -721,5 +721,28 @@ describe('Menu module', () => {
       Menu.setApplicationMenu(null)
       expect(Menu.getApplicationMenu()).to.be.null()
     })
+
+    it('provides working menu accelerators', (done) => {
```

Comment:
You'll need to conditionally skip this test if `robotjs` failed to install.

E.g. the devtools spec does this

```js
let testFn = it
try {
  // We have other tests that check if native modules work, if we fail to require
  // robotjs let's skip this test to avoid false negatives
  require('robotjs')
} catch (err) {
  testFn = it.skip
}

testFn('my test', ...
```